src-cpp/app/modules/main/wallet/accounts/view.cpp

```diff
@@ -4,14 +4,17 @@
 
 namespace Modules::Main::Wallet::Accounts
 {
-View::View(QObject* parent)
-    : QObject(parent)
+View::View(Controller *controller, QObject* parent)
+    : m_controllerPtr(controller),
+      QObject(parent)
 {
     m_modelPtr = new Model(this);
+    m_currentAccountPtr = new Item("", "", "", "", "", "", false, false, 0);
```

Comment:
just make constructor with default parameters.

Reply:
Wouldn't that affect the places that we need to call properly? like someone may miss giving some value to parameters and no error would be displayed.?